addons/hr_expense/migrations/10.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Le Filament (<https://le-filament.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+# Preserve columns which were moved to hr_expense_sheet in order to make the post migration
+column_copies = {
+    'hr_expense': [
+        ('account_move_id', None, None),
+        ('bank_journal_id', None, None),
+        ('department_id', None, None),
+        ('journal_id', None, None),
+    ],
+}
+
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.copy_columns(env.cr, column_copies)
```

Comment:
You are right, I was not sure whether the lines would not have disappeared before running post-migration, which is why I wanted to make sure they were not lost.
From your comment I understand that this is not the case so I will drop this pre-migration file.